docs/usage/config-presets.md

```diff
@@ -232,6 +232,44 @@ Parameters are supported similar to other methods:
 }
 ```
 
+## Templating presets
+
+In order to provide flexible preset configurations, Renovate provides support using [templates](./templates.md) for preset names.
+
+### Example use-case
+
+The following example shows a self-hosted Renovate setup (`config.js`) within a Git repository `my-org/renovate` of a self-hosted GitLab instance.
+Additionally, there is a shareable config preset configured within the same repository (`default.json`).
+
+Simplified repository tree
+
+```
+- .gitlab-ci.yml
+- config.js
+- default.json
+```
+
+Renovate is configured to use the shareable preset as a base configuration.
+
+```javascript
+module.export = {
+  extends: ['local>my-org/renovate'],
+};
+```
```

Comment:
It's unclear what is meant by this example. Is this meant to be the example of what not to do? Or a problem?